.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Same as above: Remove file from GitHub and add the filename to .gitignore

Reply:
Done.